open-company/open-company-web
# Remove section picker from NUX

[Trello](https://trello.com/c/Bq5kwX8t)

Review with Storage PR https://github.com/open-company/open-company-storage/pull/216

Removing section picker to streamline NUX. Urgent because there's a race condition with it right now that causes onboarding users to see the error notification flash by.

To test:

- Onboard a new org with Slack
- [x] No section picker?
- [x] Got the default sections from `config/new-org-board-names`? Including `General`?
- [x] Got the usual sample post?
- Onboard a new org with Email
- [x] No section picker?
- [x] Got the default sections from `config/new-org-board-names`? Including `General`?
- [x] Got the usual sample post?
- Invite a user
- [x] No problems onboarding?
- Setup an email domain in the org
- Onboard a user with the email domain
- [x] No problems onboarding?

Code check:

- [x] All section picker CSS gone?
- [x] All section picker code gone?

- Deploy
